firefox-ios/Client/Frontend/Microsurvey/Prompt/MicrosurveyPromptView.swift

```diff
@@ -47,12 +47,22 @@ class MicrosurveyPromptView: UIView, ThemeApplicable, Notifiable {
     private var logoWidthConstraint: NSLayoutConstraint?
     private var logoHeightConstraint: NSLayoutConstraint?
 
+    private var logoSizeScaled: CGFloat {
+        return UIFontMetrics.default.scaledValue(for: UX.logoSize)
+    }
+
     private let windowUUID: WindowUUID
     var notificationCenter: NotificationProtocol
 
     private lazy var logoImage: UIImageView = .build { imageView in
         imageView.image = UIImage(imageLiteralResourceName: ImageIdentifiers.homeHeaderLogoBall)
         imageView.contentMode = .scaleAspectFit
+        imageView.isAccessibilityElement = true
```

Comment:
What's the reasoning behind making the logo an a11y element? I thought that only images that convey information should be a11y elements - I don't think this logo counts as that.

Reply:
It conveys that this prompt is belonging to Firefox, which was a concern originally from design perspective. The note attached was `Indicates to user that prompt is coming from Firefox and not website.` While the other images in the survey are more presentational, so didn't add a11y for that.

Reply:
If this is still concern, I can follow up with a11y on this.

Reply:
ok great thank you!